blueprints/validator/files/tests/unit/validators/__name__-test.js

```diff
@@ -1,9 +1,8 @@
 import Ember from 'ember';
-import { moduleFor, test }
-from 'ember-qunit';
+import { moduleFor, test } from 'ember-qunit';
 
 moduleFor('validator:<%= dasherizedModuleName %>', 'Unit | Validator | <%= dasherizedModuleName %>', {
-  needs: ['validator:messages']
+  integration: true
```

Comment:
If you're going to make this an integration test, you should move the blueprint to `blueprints/validators/files/tests/integration/validators/__name__-test.js` to match conventions - also should probably change the description above. That said, I don't know if it needs to be an integration test.